_includes/tutorials/change-topic-partitions-replicas/ksql/markup/dev/create-new-stream.adoc

```diff
@@ -0,0 +1,7 @@
+Next, create a new ksqlDB stream—let's call it `s2`—that will be backed by a new target Kafka topic `topic2` with the desired number of partitions and replicas. Using the `WITH` clause, you can specify the partitions and replicas of the underlying Kafka topic.
+
+The result of `SELECT * FROM S1` causes every record from Kafka topic `topic1` (with 1 partition and 1 replica) to be produced to Kafka topic `topic2` (with 2 partitions and 2 replicas).
+
```

Comment:
Maybe we should add a step here instructing the user to shut down the ksqlDB cli since it's not needed beyond this step.

Reply:
@bbejeck good suggestion, adding now.